### PR TITLE
Add packaging status

### PR DESCRIPTION
**Pull request by tim77**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -99,6 +99,10 @@ install target/release/choose <DESTDIR>
 
 Just make sure DESTDIR is in your path.
 
+### Packaging status
```
Comment (theryangeary) [inline readme.md:102]: This title should read "Installing From Package Managers" and there should be another sub title between line 86 and 88 that reads "Installing From Source".

File: readme.md [hunk 2]
```diff
@@ -99,6 +99,10 @@ install target/release/choose <DESTDIR>
...
+
+Fedora/CentOS [COPR](https://copr.fedorainfracloud.org/coprs/atim/choose/): `sudo dnf copr enable atim/choose -y && sudo dnf install choose`
```
Comment (theryangeary) [inline readme.md:104]: 1. Let's get rid of `sudo`. Leave it up to the user.
2. Let's get rid of `-y`. Leave it to the user to decide these things.
3. Let's put the code in a proper code block with three backticks.
4. Split both commands on separate lines.

**Comment by theryangeary** [comment]
Hi @tim77, thanks for your interest in packaging `choose` for fedora/CentOS! 

1. You can see above I've left some specific requests about the nature of the readme. 

2. The package in COPR is a bit out of date. I realize this is because the most recent release was a bit outdated, and I've updated the releases accordingly. There is now a version `1.0.0` which I would hope to see in COPR before approving.

Thanks again for your interest in packaging `choose`!

**Comment by tim77** [comment]
Fixed. New `1.0.0` build in process...

**Pull request merged**